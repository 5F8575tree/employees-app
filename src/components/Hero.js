import React from "react";

const Hero = () => {
  return (
    <div className="hero">
      <h1 className="hero">Employees Poll</h1>
      <img className="hero" src="https://i.imgur.com/qkdpN.jpg" alt="hero" />
      <p className="hero__subtitle">
        This is a poll for employees of the company.
      </p>
    </div>
  );
};

export default Hero;
