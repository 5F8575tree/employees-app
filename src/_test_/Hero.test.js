import { render } from '@testing-library/react';
import Hero from '../components/Hero';

test('renders hero link', () => {
  render(<Hero />);
});

// we need a test using toMatchSnapshot()
test('renders hero link snapshot', () => {
  const { asFragment } = render(<Hero />);
  expect(asFragment()).toMatchSnapshot();
});
