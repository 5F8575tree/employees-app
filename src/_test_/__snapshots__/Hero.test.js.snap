// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders hero link snapshot 1`] = `
<DocumentFragment>
  <div
    class="split_screen"
  >
    <div
      class="left"
    >
      <section
        class="copy"
      >
        <h1>
          Employees Poll
        </h1>
        <img
          alt="company_logo"
          src="ktg2-final2arctic.png"
        />
        <p>
          Here's your chance to have 
          <strong>
            your
          </strong>
           say on how we work together
        </p>
      </section>
    </div>
  </div>
</DocumentFragment>
`;
